fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/PopulateResourceTypes.java

```diff
@@ -50,7 +50,7 @@ public void run(IDatabaseTranslator translator, Connection c) {
         try (Statement s = c.createStatement(); PreparedStatement batch = c.prepareStatement(stmtResourceTypeInsert)) {
             s.execute(stmtVariable);
```

Comment:
does the set variable work for derby?

Reply:
Ah, I expect not.  Main doesn't even try it because it skips `allocateTenant` when dbType == DERBY.

Reply:
How should we handle for now?  Maybe we just add an `@implNote` to say it only works for multi-tenant schemas?

Or should we overload the constructor and let the class's user tell us if its for a multi-tenant schema or not?

Reply:
@Impl note works for me

Reply:
I added the `@implNote` to the class's javadoc...maybe i should have done it on the method too, but lets call it good for now